CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog TYPO3 Crawler widget
 
+## Version 1.1.2
+
+* Added possibility to refresh the widget
```

Comment:
Please add: "... in TYPO3 v11.5" - so it is clear that it is not available in 10.

Reply:
I write my Change log based on https://keepachangelog.com/en/1.0.0/